assets/src/edit-story/components/canvas/canvasProvider.js

```diff
@@ -41,6 +41,9 @@ function CanvasProvider({ children }) {
   const [pageContainer, setPageContainer] = useState(null);
   const [fullbleedContainer, setFullbleedContainer] = useState(null);
 
+  const [isResizing, setIsResizing] = useState(false);
```

Comment:
These have certain cost so high up the editor component tree. Are these strictly necessary? It's ok, of course, if it's necessary. But pls lmk if the `useTransformHandler` can be used for this. They are targeted at a specific element being transformed.

Reply:
Will check that, but isn't that a higher provider?

![image](https://user-images.githubusercontent.com/21036224/83196652-93eb4300-a13c-11ea-8d1a-ba691ba98e09.png)

Reply:
That's great, it simplified things, thanks 🙏 
https://github.com/google/web-stories-wp/pull/1948/commits/88e3d9ade0542fec519b489798b8fcc4f9b14fab